pkg/operators/ustack/ustack.go

```diff
@@ -54,19 +55,44 @@ func (o *Operator) GlobalParams() api.Params {
 }
 
 func (o *Operator) InstanceParams() api.Params {
-	return nil
+	return api.Params{&api.Param{
+		Key:          symbolizersParam,
+		Description:  `Symbolizers to use. Possible values are: "none", "auto", or comma-separated list among: "symtab"`,
+		DefaultValue: "auto",
```

Comment:
How do we define "auto"? Would it be the same as "all" or would we differentiate? If "auto" would mean all, we could remove "none"/"auto" and just have the default value reflect all options (comma-separated). That would also make it more accessible to other frontends (e.g. we could provide "PossibleValues" and allow a set of them, so the frontend could show items you can just check/uncheck).

Reply:
At the moment, "auto" means "all", but as we add more options, I was thinking that "auto" should only include the options which are fast enough to be a reasonable default. For example, "goresym" would not be included in "auto".